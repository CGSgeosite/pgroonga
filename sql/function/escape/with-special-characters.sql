SELECT pgroonga.escape('a+O-R< >あいう~*()"'':', '()OR:');
