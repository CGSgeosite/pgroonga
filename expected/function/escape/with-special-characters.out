SELECT pgroonga.escape('a+O-R< >あいう~*()"'':', '()OR:');
           escape           
----------------------------
 a+\O-\R< >あいう~*\(\)"'\:
(1 row)

